package.json

```diff
@@ -28,7 +28,12 @@
     "search query"
   ],
   "repository": "git@github.com:mixmaxhq/search-string.git",
-  "dependencies": {},
+  "dependencies": {
+    "@mixmaxhq/toolbox": "^1.3.0",
```

Comment:
Can we make this public? `search-string` is public.. so I don't think we can depend on this.